.env

```diff
@@ -17,7 +17,9 @@ TRACKING_STRATEGY=segment
 # already exist on the host filesystem and MUST be parents of *_ROOT.
 # Issue: https://github.com/airbytehq/airbyte/issues/577
 HACK_LOCAL_ROOT_PARENT=/tmp
-WEBAPP_URL=http://localhost:8000/
+WEBAPP_PORT=8000
+SERVER_PORT=8001
+WEBAPP_URL=http://localhost:${WEBAPP_PORT}/
 API_URL=/api/v1/
 TEMPORAL_HOST=airbyte-temporal:7233
-INTERNAL_API_HOST=airbyte-server:8001
+INTERNAL_API_HOST=airbyte-server:${SERVER_PORT}
```

Comment:
does variable interpolation work in the .env file?

How does this change work with @jrhizor modification to server everything from a single port?